.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,16 @@
-Thanks you for your contribution to the Azure-SDK-for-Go! We will triage and review it as quickly as we can.
+<!--
+Thank you for contributing to the Azure SDK for Go.
 
-As part of your submission, please make sure that you can make the following assertions:
+Please verify the following before submitting your PR, thank you!
+-->
 
- - [ ] I'm not making changes to Auto-Generated files which will just get erased next time there's a release.
-   - If that's what you want to do, consider making a contribution here: https://github.com/Azure/autorest.go
- - [ ] I've tested my changes, adding unit tests where applicable.
- - [ ] I've added Apache 2.0 Headers to the top of any new source files.
- - [ ] I'm submitting this PR to the `dev` branch, or I'm fixing a bug that warrants its own release and I'm targeting the `master` branch.
- - [ ] If I'm targeting the `master` branch, I've also added a note to [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-go/blob/master/README.md).
- - [ ] I've mentioned any relevant open issues in this PR, making clear the context for the contribution.
- 
\ No newline at end of file
+- [ ] The purpose of this PR is explained in this or a referenced issue.
+- [ ] The PR does not update generated files.
+   - These files are managed by the codegen framework at [Azure/autorest.go][].
+- [ ] The PR targets the `latest` branch.
+- [ ] Tests are included and/or updated for code changes.
+- [ ] Updates to [CHANGELOG.md][] are included.
+- [ ] Apache v2 license headers are included in each file.
```

Comment:
[off-topic] what would it take for us to change our license to MIT?

Reply:
Lots of time with the legal team :). Let's discuss motivations for this offline.